scripts/covid_hosp_prod.Rmd

```diff
@@ -0,0 +1,65 @@
+---
+title: COVID Forecaster Predictions
+author: COVID Forecast Team
+date: "Rendered: `r format(Sys.time(), '%d %B %Y')`"
+output:
+  html_document:
+    toc: True
+    # self_contained: False
+    # lib_dir: libs
+params:
+  forecast_generation_date: !r Sys.Date()
+  forecast: !r ""
+---
+
+```{css, echo=FALSE}
+body {
+  display: block;
+  max-width: 1280px !important;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+body .main-container {
+  max-width: 1280px !important;
+  width: 1280px !important;
+}
+```
+
+```{r setup, include=FALSE}
+library(dplyr)
+library(magrittr)
+source(here::here("scripts", "plotting.R"))
+
+exclude_geos <- c("as", "gu", "mp", "vi")
```

Comment:
it appears to be excluding the hhs timeseries but not the prediction.

Reply:
![image](https://github.com/cmu-delphi/exploration-tooling/assets/7095649/32c11c42-3138-4b2b-aa3b-86a05698adbe)